src/auth/dto/change-password-request.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString, MinLength } from 'class-validator';
+
+export class ChangePasswordRequest {
+  @MinLength(12)
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty()
+  newPassword: string;
```

Comment:
have you got any requirements for the password (escape of length) maybe you should use `@ValidatePassword()`